.github/workflows/code-review.yml

```diff
@@ -0,0 +1,13 @@
+on: [pull_request]
+
```

Comment:
If we provide a name attribute here (ie https://github.com/slackhq/kaldb/blob/master/.github/workflows/maven.yml) it makes the PR tests a little easier to understand - otherwise it just uses the file path.

Reply:
![image](https://github.com/slackhq/kaldb/assets/771133/c0e24cf2-390e-4e66-bf7d-95a88d0aaced)

Reply:
Done